examples/step-1/doc/intro.dox

```diff
@@ -137,22 +137,47 @@ later on. Additionally, as of deal.II version 9.0, GridGenerator::hyper_shell()
 now automatically attaches a SphericalManifold to the Triangulation. Otherwise
 the rest of the lecture material is relevant.
 
+
 <h3> What this program does </h3>
 
-Let's come back to step-1, the current program.
-In this first example, we don't actually do very much, but show two
-techniques: what is the syntax to generate triangulation objects, and
-some elements of simple loops over all cells. We create two grids, one
-which is a regularly refined square (not very exciting, but a common
-starting grid for some problems), and one more geometric attempt: a
-ring-shaped domain, which is refined towards the inner edge. Through
-this, you will get to know three things every finite element program
-will have to have somewhere: An object of type Triangulation for the
-mesh; a call to the GridGenerator functions to generate a mesh; and
-loops over all cells that involve iterators (iterators are a
-generalization of pointers and are frequently used in the C++ standard
-library; in the context of deal.II, the @ref Iterators module talks
-about them).
+Let's come back to step-1, the current program. The goal of this
+program is to introduce you to the Triangulation class that is at the
+core of every finite element program. The name "triangulation" in this
+context is mostly historical: To finite element practitioner, the
+terms "triangulation", "mesh", and "grid" are all synonymous and
+describe a subdivision of a domain on which a differential equation is
+posed into cells of some kind. If the domain is two-dimensional, these
+cells may indeed be triangles, but they could also be quadrilaterals
+(four-sided objects such as squares and rectangles, and their
+deformations). In one space dimension, the cells are line segments. In
+three space dimensions, they can be tetrahedra, hexahedra (deformed
+cubes), pyramids (a four-sided base with three triangles connecting to
+a point at the top), and "wedges" (two triangles at the bottom and
+top, connected by three quadrilaterals). Collections of any such cells are
+"triangulations" in common usage of the word even though they may not
+actually have triangles in them. All of them are also "grids" in
+common usage of the word even though the vertices are in neat rows
+parallel to the coordinate axes. In other words, whenever you read any
```

Comment:
Here I do not entirely follow the text. Do you want to say that the classical definition of a grid has the vertices "neat rows parallel to the coordinate axes", but the term is also used for general point clouds? I think one understands what you would like to express, it's just that I don't quite follow with the logic of the text here.

Reply:
Yes, I think of the term "grid" as in Karopapier. I added another half sentence here, and mentioned the term "structured grid".